src/progress-indicator/progress-indicator.tsx

```diff
@@ -15,15 +16,15 @@ export const ProgressIndicator = <T,>({
     steps,
     currentIndex,
     displayExtractor,
-    fadeColor,
-    fadePosition = "both",
```

Comment:
can revert this? it is intended so that the extra props don't get passed down to the underlying div element

rename to `_fadeColor` to avoid the eslint warning